metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -15,12 +15,8 @@ to collect. The example below collects processor time and disk writes.
   metricsets: ["perfmon"]
   period: 10s
   perfmon.counters:
-    - group: "processor"
-      collectors:
-        - alias: "processor_time"
-          query: '\Processor Information(_Total)\% Processor Time'
-    - group: "disk"
-      collectors:
-        - alias: "bytes_written"
-          query: '\PhysicalDisk(_Total)\Disk Writes/sec'
+    - alias: "processor.processor_time"
+      query: '\Processor Information(_Total)\% Processor Time'
+    - alias: "disk.bytes_written"
```

Comment:
How about `diskio.write.bytes`? More closely matches the [diskio](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-system-diskio.html) metricset.